packages/openapi3/src/openapi.ts

```diff
@@ -1409,6 +1409,10 @@ function createOAPIEmitter(
     switch (parameter.type) {
       case "header":
         return mapHeaderParameterFormat(parameter);
+      case "cookie":
+        // style and explode options are omitted from cookies
+        // https://github.com/microsoft/typespec/pull/4761#discussion_r1803365689
+        return {};
```

Comment:
because the default is different we might still want to mark `explode: false` for openapi cookies no?

Reply:
Sorry, I still don't understand the relationship between explode in TypeSpec and OpenAPI.
Is explode=true the default in OpenAPI and the default we want is explode=false?

Reply:
The OpenAPI explode option is complicated and I don't understand it, but at least it seems that explode: true is not the behavior we want. I specified explode: false. Thanks!

Reply:
yeah I don't really get the point of having it in openapi3 tbh but having `explode: true` just seems too limiting. Maybe it was just added for consistency with header and query.

So I think your current change is what we'd want.